module/Company/src/Company/Model/Company.php

```diff
@@ -112,94 +98,95 @@ public function getId()
     {
         return $this->id;
     }
-
-    public function getAsciiName()
+    
+    /**
+     * Get the company's translations.
+     *
+     * @return array
+     */
+    public function getTranslations()
     {
-        return $this->asciiName;
+        return $this->translations;
     }
 
-    public function setAsciiName($asciiName)
-    {
-        $this->asciiName = $asciiName;
-    }
     /**
-     * Get the company's name.
+     * Add a translation.
      *
-     * @return string
+     * @param CompanyI18n $translation
      */
-    public function getName()
+    public function addTranslation(CompanyI18n $translation)
     {
-        return $this->name;
+        $this->translations->add($translation);
     }
-
+    
     /**
-     * Set the company's name.
-     *
-     * @param string $name
+     * Remove a translation.
+     * 
+     * @param CompanyI18n $translation Translation to remove
      */
-    public function setName($name)
+    public function removeTranslation(CompanyI18n $translation) 
     {
-        $this->name = $name;
+        $this->translations->removeElement($translation);
     }
-
+    
     /**
-     * Get the company's address.
+     * Get the company's name.
      *
      * @return string
      */
-    public function getAddress()
+    public function getName()
     {
-        return $this->address;
+        return $this->name;
     }
 
     /**
-     * Set the company's address.
+     * Set the company's name.
      *
-     * @param string $address
+     * @param string $name
      */
-    public function setAddress($address)
+    public function setName($name)
     {
-        $this->address = $address;
+        $this->name = $name;
     }
 
     /**
-     * Get the company's website.
-     *
-     * @return string
+     * Gets the company's ascii name.
+     * 
+     * @return string the company's ascii name
      */
-    public function getWebsite()
+    public function getAsciiName()
     {
-        return $this->website;
+        return $this->asciiName;
     }
```

Comment:
De asciiName is eigenlijk een slug. Dus een URL-vriendelijke naam voor het bedrijf. De naam asciiName is ongelukkig gekozen aangezien ook niet alle ascii karakters url vriendelijk zijn. Wij overwegen om overal asciiName te wijzigen naar slug.